datacite-delete-draft-doi-handler/src/test/java/no/unit/nva/datacite/handlers/doi/DeleteDraftDoiHandlerTest.java

```diff
@@ -39,8 +40,6 @@
 @WireMockTest(httpsEnabled = true)
 public class DeleteDraftDoiHandlerTest {
 
-    public static final String DRAFT_STATE = "draft";
-    public static final String PUBLISHED_STATE = "PUBLISHED";
```

Comment:
Creating doi state Enum. We do not want different strings everywhere in the project.